Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:5.1
+import PackageDescription
+
+let package = Package(
+    name: "CodableFirebase",
+    products: [
+        .library(name: "CodableFirebase", targets: ["CodableFirebase"])
+    ],
+    targets: [
+        .target(
+            name: "CodableFirebase",
+            path: "CodableFirebase"
```

Comment:
Isn't this path infered?

Reply:
I believe the inferred path is Sources/<TargetName>. Not sure if it will infer <TargetName> at root.